models/intel/bert-small-uncased-whole-word-masking-squad-0002/description/bert-small-uncased-whole-word-masking-squad-0002.md

```diff
@@ -0,0 +1,75 @@
+# bert-small-uncased-whole-word-masking-squad-0002
+
+## Use Case and High-Level Description
+
+This is a small BERT-large like model distilled on SQuAD v1.1 training set from original
+bert-large-uncased-whole-word-masking-finetuned-squad provided by the [Transformers](https://github.com/huggingface/transformers) library.
+The model performs question answering for English language;
+the input is a concatenated premise and question for the premise,
+and the output is the location of the answer to the question inside the premise.
+For details about the original model, check out
+[BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding](https://arxiv.org/abs/1810.04805),
+[HuggingFace's Transformers: State-of-the-art Natural Language Processing](https://arxiv.org/abs/1910.03771).
+
+Tokenization occurs using the BERT tokenizer (see the demo code for implementation details) and the enclosed `vocab.txt` dictionary file. Input is to be lower-cased before tokenizing.
+
+## Specification
+
+| Metric            | Value                 |
+|-------------------|-----------------------|
+| GOps              | 23.9                  |
+| MParams           | 41.1                  |
+| Source framework  | PyTorch\*             |
+
+## Accuracy
+
+The quality metrics were calculated on the SQuAD v1.1 dataset ("dev" split). Maximum sequence length is 384, input is lower-cased.
+
+| Metric                    | Value         |
+|---------------------------|---------------|
+| F1                        |        91.9%  |
+| Exact match (EM)          |        85.4%  |
+
+## Performance
+
+## Input
+
+1. Token IDs, name: `input_ids`, shape: [1x384].
+Token IDs is sequence of integer values that is representing the tokenized premise and question.
+The sequence structure is as follows (`[CLS]`, `[SEP]` and `[PAD]` should be replaced by corresponding token IDs
+as specified by the dictionary):
+`[CLS]` + *tokenized question* + `[SEP]` + *tokenized premise of the question* + `[SEP]` + (`[PAD]` tokens to pad to the maximum sequence length of 384)
+
+2. Input mask, name: `attention_mask`, shape: [1x384].
+Input mask is a sequence of integer values representing the mask of valid values in the input.
+The values of this input are equal to:
+    * `1` at positions corresponding to the `[CLS]` + *tokenized question* + `[SEP]` + *tokenized premise of the question* + `[SEP]` part of the `input_ids`  (i.e. all positions except those containing the `[PAD]` tokens), and
+    * `0` at all other positions
+
+3. Token types,  name: `token_type_ids`, shape: [1x384].
+Token types is sequence of integer values representing the segmentation of the `input_ids` into question and premise.
```

Comment:
'premise'? maybe 'context'?

Reply:
I agree to use any of them but this word (premise) is used in description of all other bert models. if I change it here it will be inconsistent. should we change it in each bert model description? 
add @vshampor to discuss this.

Reply:
https://dictionary.cambridge.org/dictionary/english/premise

Personally I'm ok with "context" as well, as long as it is also used consistently across all descriptions.